x-pack/platform/plugins/shared/alerting/server/routes/gaps/apis/get_rule_ids_with_gaps/get_rule_ids_with_gaps_route.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { IRouter } from '@kbn/core/server';
+import {
+  getRuleIdsWithGapQuerySchemaV1,
+  GetRuleIdsWithGapQueryV1,
+  GetRuleIdsWithGapResponseV1,
+} from '../../../../../common/routes/gaps/apis/get_rules_with_gaps';
+import { ILicenseState } from '../../../../lib';
+import { verifyAccessAndContext } from '../../../lib';
+import {
+  AlertingRequestHandlerContext,
+  INTERNAL_ALERTING_GAPS_GET_RULES_API_PATH,
+} from '../../../../types';
+import { transformResponseV1 } from './transforms';
+
+export const getRuleIdsWithGapsRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.post(
+    {
+      path: `${INTERNAL_ALERTING_GAPS_GET_RULES_API_PATH}`,
+      validate: {
+        body: getRuleIdsWithGapQuerySchemaV1,
```

Comment:
can we rename this `getRuleIdsWithGapBodySchemaV1` since it is the schema for the request body?

Reply:
https://github.com/elastic/kibana/commit/aa510df65bec67b01811160d05132a13c2793594